FlowCrypt/Lib/GMP/gmp-interface.m

```diff
@@ -0,0 +1,27 @@
+#include "gmp.h"
+#include <stdio.h>
+
+const char* c_gmp_mod_pow(const char* base, const char* exponent, const char* modulo) {
+#ifdef __arm64__
```

Comment:
@sosnovsky you mentioned making this work for other platforms (simulator, presumably amd64). Should this be updated to also allow this method for whatever other platforms that you built for?

Reply:
Yes, I'll remove this check, as now GMP library should work on all platforms - devices and simulators.